### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined while attempting to toggle Omnisharp

### ISSUE THREAD

**Issue by ronrat** [OmniSharp/omnisharp-atom#214]
1. Opened folder containing Visual Studio solution.
2. Use command palette to execute OmniSharp Atom: Toggle command.

**Atom Version**: 0.196.0
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.4.12

### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omni-sharp-server\client-manager.ts:50

TypeError: Cannot read property 'split' of undefined
  at ClientManager.getClientForEditor (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omni-sharp-server\client-manager.ts:50:33)
  at ClientManager.getClientForActiveEditor (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omni-sharp-server\client-manager.ts:39:25)
  at Function.Omni.toggle (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omni-sharp-server\omni.ts:15:24)
  at OmniSharpAtom.toggle (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:145:18)
  at atom-workspace.<anonymous> (C:\Users\rratzlaff\.atom\packages\omnisharp-atom\lib\omnisharp-atom\omnisharp-atom.ts:44:97)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\command-registry.js:238:29)
  at C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\command-palette\lib\command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\atom-space-pen-views\lib\select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\atom-space-pen-views\lib\select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\command-registry.js:238:29)
  at C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (C:\Users\rratzlaff\AppData\Local\atom\app-0.196.0\resources\app.asar\src\window-event-handler.js:178:20)

```

### Commands

```
     -0:39.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:39.4.0 editor:move-to-end-of-word (atom-text-editor.editor.is-focused)
     -0:32.9.0 application:add-project-folder (atom-text-editor.editor.is-focused)
     -0:21.4.0 omnisharp-atom:toggle-output (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:19 command-palette:toggle (atom-text-editor.editor.is-focused)
  3x -0:16 core:move-down (atom-text-editor.editor.mini)
     -0:12.9.0 core:confirm (atom-text-editor.editor.mini)
     -0:12.9.0 omnisharp-atom:show-omni (atom-text-editor.editor.is-focused)
     -0:09.4.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:03.6.0 core:move-down (atom-text-editor.editor.mini)
     -0:02.5.0 core:confirm (atom-text-editor.editor.mini)
     -0:02.5.0 omnisharp-atom:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autocomplete",
      "autocomplete-plus-async"
    ],
    "autoHideMenuBar": true,
    "projectHome": "c:\\code"
  }
}
```

### Installed Packages

```coffee
# User
atom-yeoman, v0.2.0
autocomplete-plus, v2.12.1
color-picker, v1.7.0
jsonlint, v1.1.1
language-csharp, v0.5.0
language-mediawiki, v0.1.0
language-powershell, v2.0.1
language-typescript, v0.15.0
linter, v0.12.1
minimap, v4.8.0
omnisharp-atom, v0.4.12
project-manager, v1.15.5

# Dev
No dev packages
```

**Comment by ladislaolainez** [comment]
Same bug here. Previous version worked just fine.

**Comment by nosami** [comment]
fixed in v0.4.21

**Comment by wtilton** [comment]
This is happening to me today, after installing this yesterday. :( No clue.
